docs/release_notes.md

```diff
@@ -1,5 +1,82 @@
 #
 
+## 4.3.3
+
+Release Date: 2024-08-26
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 4.3.3.
+
+This release focuses mostly on adding support for newer compilers, libstdc++, Ubuntu 24.04, and PostgreSQL 15 and newer.
+
+Other notable work includes a fix for a msiServerMonPerf vulnerability and more GenQuery2 readiness before it is promoted to the default query parser in a later release.
+
+Additionally, the package revision and filenames of the plugins have been updated to include the server version they were built against (e.g. irods-rule-engine-plugin-logical-quotas_4.3.3.0-0+4.3.3~jammy_amd64.deb).  Package names and dependencies have not changed.  This semi-decoupling will allow a plugin to be rebuilt for later releases, without having to bump its own version number if no changes have been made to the plugin itself.  This change will affect any local scripts that may be used to install and maintain iRODS installations and deployments.
```

Comment:
Is it worth mentioning *how* this affects the package version? For instance, today in order to install 4.3.2 in particular, one must type (something like) the following for `apt` on Ubuntu 20.04:
```bash
sudo apt install irods-rule-engine-plugin-logical-quotas==4.3.2.0-0~focal
```
I think it goes something like this for 4.3.3:
```bash
sudo apt install irods-rule-engine-plugin-logical-quotas==4.3.3.0-0+4.3.3~focal
```

Reply:
I'm basing this off of the `dpkg -I` output from my locally built package for 4.3.3 testing (notice the **Version** line):
```
 new Debian package, version 2.0.
 size 271670 bytes: control archive=1167 bytes.
    1467 bytes,    18 lines      control              
     224 bytes,     2 lines      md5sums              
     917 bytes,    15 lines   *  postinst             
 Architecture: amd64
 Breaks: irods-rule-engine-plugin-logical-quotas
 Depends: irods-externals-avro-libcxx1.11.0-3, irods-externals-boost-libcxx1.81.0-1, irods-externals-fmt-libcxx8.1.1-1, irods-externals-nanodbc-libcxx2.13.0-2, irods-externals-zeromq4-1-libcxx4.1.8-1, irods-externals-spdlog-libcxx1.9.2-2, irods-externals-libarchive3.5.2-0, irods-externals-clang-runtime13.0.1-0, irods-server (= 4.3.3), irods-runtime (= 4.3.3), libc6
 Description: The integrated Rule-Oriented Data System
  The Integrated Rule-Oriented Data System (iRODS) is open source data management software used by research, commercial, and governmental organizations worldwide.
  .
  iRODS is released as a production-level distribution aimed at deployment in mission critical environments. It virtualizes data storage resources, so users can take control of their data, regardless of where and on what device the data is stored.
  .
  The development infrastructure supports exhaustive testing on supported platforms; plugin support for microservices, storage resources, authentication mechanisms, network protocols, rule engines, new API endpoints, and databases; and extensive documentation, training, and support services.
 Homepage: https://irods.org
 Maintainer: iRODS Consortium <packages@irods.org>
 Package: irods-rule-engine-plugin-logical-quotas
 Priority: extra
 Replaces: irods-rule-engine-plugin-logical-quotas
 Section: contrib/science
 Version: 4.3.3.0-0+4.3.3~focal
 Installed-Size: 1002
```

Reply:
that is correct.    perhaps this goes on the installation page, rather than the release notes page... can discush.

Reply:
I think there's value in documenting this since it's a change that affects users managing scripts that install iRODS.

We need to confirm the correct way to target a specific version first though.

Reply:
I agree. The installation page sounds like a good place for that.